services/dynamic_addressbook_add_node.proto

```diff
@@ -0,0 +1,79 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2018 - 2021 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+option java_package = "com.hederahashgraph.api.proto.java";
+// <<<pbj.java_package = "com.hedera.hapi.node.consensus">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+
+import "basic_types.proto";
+
+/**
+ * See [DynamicAddressBookService.addNode()](#proto.DynamicAddressBookService)
+ */
+message DynamicAddressBookAddNodeTransactionBody {
+    /**
+     * The unique entity id of the Node.
+     */
+    int64 node_id = 1;
```

Comment:
Aren't we using CamelCase in the protobuf definitions?

Reply:
Updated to CamelCase.

Reply:
No, we use snake_case as is [recommended](https://protobuf.dev/programming-guides/style/#message-field-names) by Google.

Reply:
I wonder if I look into the wrong repository. I can see some snake_case here and there, but most of the files I have looked at use CamelCase.

Reply:
We did a bulk cleanup in the [past](https://github.com/hashgraph/hedera-protobufs/pull/13). Probably we need to do another.

Reply:
Changed back to snake-case.